src/data/finset/lattice.lean

```diff
@@ -429,6 +430,15 @@ lemma inf_sup_distrib_right (s : finset ι) (f : ι → α) (a : α) :
   s.inf f ⊔ a = s.inf (λ i, f i ⊔ a) :=
 @sup_inf_distrib_right αᵒᵈ _ _ _ _ _ _
 
+protected lemma codisjoint_inf_right : codisjoint a (s.inf f) ↔ ∀ i ∈ s, codisjoint a (f i) :=
+by simp only [codisjoint_iff, inf_sup_distrib_left, finset.inf_eq_top_iff]
+
+protected lemma codisjoint_inf_left : codisjoint (s.inf f) a ↔ ∀ i ∈ s, codisjoint (f i) a :=
+by simp only [codisjoint_iff, inf_sup_distrib_right, finset.inf_eq_top_iff]
```

Comment:
Don't these follow by dualizing?

Reply:
Sure, but given the miserable status of `OrderDual` in mathlib4, it's probably better to keep the explicit proofs.